public/content/contact.mdx

```diff
@@ -102,3 +106,38 @@
 </Accordion>
 
 </Section>
+<Section>
+
+# Contact
+
+<Accordion title="Licensing inquiry" preserveInnerState={true}>
+  <p>
```

Comment:
Does this actually require ``<p></p>`` tags given that it's mdx?

Reply:
Oh, I suppose not. If I put blank lines at the start and end of the `<Accordion>` content it would be parsed as markdown.